prow/cluster/starter.yaml

```diff
@@ -0,0 +1,543 @@
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
install-prow.sh should now apply this file, not from ```https://raw.githubusercontent.com/kubernetes/test-infra/a202e595a33ac92ab503f913f2d710efabd3de21/prow/cluster/starter.yaml```

Reply:
Done